tests/e2e/test_e2e.py

```diff
@@ -76,9 +77,10 @@ def test_e2e():
     PARALLEL = 2
 
     max_docs = 12
+    dirpath = "temp_test_e2e"
```

Comment:
Looks like by default log_dir is tempfile.gettempdir(). Do, rather than specifying log_folder, you can just let it go to the temp directory and not worry about clean-up.